project/SnackfsBuild.scala

```diff
@@ -24,42 +24,15 @@ object SnackfsBuild extends Build {
 
   lazy val VERSION = "0.6.1-EA"
 
-  lazy val CAS_VERSION = "1.2.12"
+  lazy val CAS_VERSION = "2.1.0"
 
-  lazy val THRIFT_VERSION = "0.7.0"
+  lazy val THRIFT_VERSION = "0.9.1"
 
   lazy val TWITTER_UTIL_VERSION = "6.7.0"
 
   lazy val dist = TaskKey[Unit]("dist", "Generates project distribution")
 
-  lazy val pom = {
```

Comment:
We need to keep the POM section intact. Please add your name and details to the developers section here.